apps/server/src/shared/repo/school/school.repo.ts

```diff
@@ -7,4 +7,10 @@ export class SchoolRepo extends BaseRepo<School> {
 	get entityName() {
 		return School;
 	}
+
+	async createAndSave(entity: School): Promise<School> {

```

Comment:
Is there something like a transactional thing in mikroorm?
Here we do both because we want the entity back because we need the id of the school to save the user later. If we would have a unique identifier (no id) for schools we wouldnt need the id.
We gave that to alex.

Reply:
Haven't seen any use of explicit transaction management or locking so far anywhere. Since school is a cross-domain entity, and you need the reference, I'd flush right away. The scope of this feature is updating / creating a single user (with its school) anyway, right?

Reply:
Haven't seen any use of explicit transaction management or locking so far anywhere. Since school is a cross-domain entity, and you need the reference, I'd flush right away. The scope of this feature is updating / creating a single user (with its school) anyway, right?

Reply:
Haven't seen any use of explicit transaction management or locking so far anywhere. Since school is a cross-domain entity, and you need the reference, I'd flush right away. The scope of this feature is updating / creating a single user (with its school) anyway, right?